tests/src/test/kotlin/dev/restate/e2e/WorkflowAPITest.kt

```diff
@@ -67,7 +62,6 @@ class JavaWorkflowAPITest {
 }
 
 @Tag("always-suspending")
-@Disabled("node-services is not ready with the new interfaces")
 class NodeWorkflowAPITest {
```

Comment:
If the test code is the same as the java test can you make a base class with the test method and the subclasses with the deployer extensions? (just look how it's done in all the other test runners)